src/Symfony/Component/Intl/DateFormatter/IntlDateFormatter.php

```diff
@@ -194,11 +194,15 @@ public function format($timestamp)
     {
         // intl allows timestamps to be passed as arrays - we don't
         if (\is_array($timestamp)) {
-            $message = 'Only integer Unix timestamps and DateTime objects are supported';
+            $message = 'Only Unix timestamps and DateTime objects are supported';
 
             throw new MethodArgumentValueNotImplementedException(__METHOD__, 'timestamp', $timestamp, $message);
         }
 
+        if (\is_string($timestamp) && $dt = \DateTime::createFromFormat('U', $timestamp)) {
```

Comment:
This should also be applied to the polyfills repo, right ?

Reply:
yep, see https://github.com/symfony/polyfill/pull/357